cleverhans/compat.py

```diff
@@ -23,9 +23,9 @@ def reduce_function(op_func, input_tensor, axis=None, keepdims=None,
 
     if LooseVersion(tf.__version__) < LooseVersion('1.8.0'):
         warning = "Running on tensorflow version " + \
-                   LooseVersion(tf.__version__).vstring + \
-                   ". This version will not be supported by CleverHans" + \
-                   "in the future."
+            LooseVersion(tf.__version__).vstring + \
+            ". This version will not be supported by CleverHans" + \
+            "in the future."
```

Comment:
Can we change this to "Support for this version in CleverHans is deprecated and may be removed on or after 2019-01-26"?